### DISCUSSION TITLE
Negotiation declining after confirmation.

### DISCUSSION THREAD

**Discussion by ndr-brt** [eclipse-edc/Connector#2260] [Q&A]
What should happen to a negotiation declined after its confirmation?
Should the agreement be removed from the negotiation?

Because currently if an agreement exists, also after a declination, the transfer can succeed (see [ContractValidationRule](https://github.com/eclipse-edc/Connector/blob/c2bc86f0e4eafbf925e145b69ef5de9d3cd0a359/extensions/control-plane/data-plane-transfer/data-plane-transfer-sync/src/main/java/org/eclipse/edc/connector/dataplane/transfer/sync/validation/ContractValidationRule.java)).

**Comment by jimmarino** [comment]
This is also will be addressed in the new IDS spec version.

**Reply by ndr-brt** [reply]
Would it make sense to define an issue for this and label it as `ids-next` or something?
